sites/frontend/components/Footer/index.js

```diff
@@ -0,0 +1,235 @@
+        },
+        {
+            title: 'Subscribe',
+            url:
+                'https://support.theguardian.com/subscribe?INTCMP=footer_support_subscribe&acquisitionData=%7B%22source%22%3A%22GUARDIAN_WEB%22%2C%22componentType%22%3A%22ACQUISITIONS_FOOTER%22%2C%22componentId%22%3A%22footer_support_subscribe%22%2C%22referrerPageviewId%22%3A%22jjztcoahsd3lnjj62kb8%22%2C%22referrerUrl%22%3A%22https%3A%2F%2Fwww.theguardian.com%2Fbooks%2F2018%2Fjul%2F24%2Fman-booker-prize-2018-longlist-nick-drnaso-sabrina-ondaatje-graphic-novel%22%7D',
+        },
+        {
+            title: 'Guardian Labs',
+            url: '/guardian-labs',
+        },
+    ],
+    [
+        {
+            title: 'About us',
+            url: '/about',
+        },
+        {
+            title: 'Work for us',
+            url: 'https://workforus.theguardian.com/locations/london',
+        },
+        {
+            title: 'Advertise with us',
+            url: 'https://advertising.theguardian.com/',
+        },
+        {
+            title: 'Contact us',
+            url: '/help/contact-us',
+        },
+        {
+            title: 'Help',
+            url: '/help',
+        },
+    ],
+    [
+        {
+            title: 'Terms & conditions',
+            url: '/help/terms-of-service',
+        },
+        {
+            title: 'Privacy policy',
+            url: '/info/privacy',
+        },
+        {
+            title: 'Cookie policy',
+            url: '/info/cookies',
+        },
+        {
+            title: 'Securedrop',
+            url: 'https://securedrop.theguardian.com/',
+        },
+        {
+            title: 'Digital newspaper archive',
+            url: 'https://theguardian.newspapers.com/',
+        },
+        {
+            title: 'Complaints and corrections',
+            url: '/info/complaints-and-corrections',
+        },
+    ],
+    [
+        {
+            title: 'All topics',
+            url: '/index/subjects/a',
+        },
+        {
+            title: 'All contributors',
+            url: '/index/contributors',
+        },
+        {
+            title: 'Modern Slavery Act',
+            url:
+                '/info/2016/jul/27/modern-slavery-and-our-supply-chains?INTCMP=NGW_FOOTER_UK_GU_MODERN_SLAVERY_ACT',
+        },
+        {
+            title: 'Facebook',
+            url: 'https://www.facebook.com/theguardian',
+        },
+        {
+            title: 'Twitter',
+            url: 'https://twitter.com/guardian',
+        },
+    ],
+];
+
+const Footer = styled('footer')({
+    backgroundColor: '#333',
+    color: '#dcdcdc',
+    fontFamily: textSans,
+    fontSize: '14px',
+
+    a: {
+        color: '#dcdcdc',
+        textDecoration: 'none',
+    },
+
+    '.footer-links': {
```

Comment:
I'm not sure about mixing classes into Emotion. It's less efficient than letting Emotion handle styling all by itself. Also adds a bit of indirection, and creates coupling between CSS and JSX structure. What do you think about extracting these into separate components?

Reply:
Hey, yeah so that is an interesting question! I'll put my thoughts in the dropdown PR as I agree there are some pros/cons to each approach.

Reply:
Cool, let's put a pin in this for now, we'll work on a code style guide when you're back